Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7-slim-buster
+
+COPY . /work
+WORKDIR /work
+
+RUN cp config.dev.py config.py
+
+RUN python3 -m venv /opt/venv
```

Comment:
the virtual env probably isnt needed within a docker.

I originally put it in when I was having trouble with the https issue detailed later, but I didn't remove it since it keeps the test env as close as possible to local.